### PR TITLE
KM408 👌 Improve cluster yaml helptexts

### PR DESCRIPTION
**Pull request by deifyed**
## Description

- Shortens `scaffold cluster` template helptexts
- Comments out the whole integrations section due to being optional
- Removes VPC section
- Adds attribute reference page on okctl.io for cluster.yaml

## Motivation and Context

[KM408](https://trello.com/c/iNLnD14G)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I have updated the release notes (for the next release).

### CODE REVIEW

File: pkg/commands/scaffoldcluster.go [hunk 1]
```diff
@@ -44,102 +44,81 @@ func sanitizeOpts(opts ScaffoldClusterOpts) ScaffoldClusterOpts {
...
+#  # Enable Promtail, feeding pod logs to Loki
+#  promtail: true
+#
+#  # Enable Loki, exposing pod logs in Grafana
+#  loki: true
+#
+#  # Enable Tempo, exposing traces in Grafana
+#  tempo: true
+
+## Specify the database(s) for okctl to provision
```
Comment (yngvark) [inline pkg/commands/scaffoldcluster.go:114]: I thought this was one long sentence.

I'm tried making more separation to clearly see that "namespace" and "user" is attributes, but I wasn't able to make something I was happy with.

```suggestion
## Specify the database(s) for okctl to provision.
## namespace is for placing database information
## user defines the admin user to provision for operations
```

**Review by yngvark — APPROVED** [review]
Sweet.

**Review by ivaruf — APPROVED** [review]
This looks very nice. Great work!

File: pkg/commands/scaffoldcluster.go [hunk 2]
```diff
@@ -44,102 +44,81 @@ func sanitizeOpts(opts ScaffoldClusterOpts) ScaffoldClusterOpts {
...
-## Name defines the name of the database to provision
 #  - name: dbname
-## Namespace defines which namespace to place the database information (secret for user, pass and configmap hostname, port
 #    namespace: relevantnamespace
-## User defines which user to provision for operations
 #    user: postgres
 
-## okctl creates a Virtual Private Cloud for you which it organizes all the intended resources that require networking.
-## A VPC is mandatory, but can be configured by the following attributes.
-#vpc:
```
[the remark this answers is not in the document]
Comment (deifyed) [reply pkg/commands/scaffoldcluster.go:138]: Yeah definitely. We still want the functionality, just don't want to overwhelm our most fragile users. The VPC configuration is defined in the cluster manifest reference

**Pull request merged**